.binder/environment.yml

```diff
@@ -3,4 +3,5 @@ channels:
   - conda-forge
 dependencies:
   - parcels
+  - pooch
```

Comment:
Don't forget to also update the condo-forge recipe?

Reply:
Oh yes!